### PR TITLE
layout: Added multi sub-leaf fields

### PR DESCRIPTION
**Pull request by andrewjj20**
The Structured Extended Flags are the first ones added for this.

### CODE REVIEW

File: src/config.yaml [hunk 1]
```diff
@@ -39,6 +39,33 @@ cpuids:
           name: vme
           bit: 0
 
+  0x00000007:
+    name: "Structured Extened Flags"
+    data_type:
+      type: SubLeafBitField
+      leaves:
+        - eax:
```
Comment (cmouzaoui) [inline src/config.yaml:47]: what does it mean for there to be 3 of each register?

Comment (andrewjj20) [reply src/config.yaml:47]: This is a list of maps. each map is a struct that has fields eax,ebx,ecx,edx that are also lists.

Comment (andrewjj20) [reply src/config.yaml:47]: in this case, it means there could be 3 sub-leaves.

Comment (andrewjj20) [reply src/config.yaml:47]: it is probably worth looking at the `BitField` structure that those end up becoming.

**Review by cmouzaoui — APPROVED** [review]
I'll assume this leads to the correct output!

**Pull request merged**